docs/guides/create_service_existing_agent.md

```diff
@@ -1 +1,212 @@
+
+##What you will learn
+In this guide, we will show how to:
+
+  - Create a service definition using an already available agent on the  [IPFS](https://ipfs.io/).
+  - Test the service using a local deployment.
+  - Publish the service on the [IPFS](https://ipfs.io/).
+  - Register the service in the on-chain protocol. We will be using the [Görli testnet](https://goerli.net/).
+  - Deploy the registered service.
+
+If you recall the [overview of the development process](./overview_of_the_development_process.md), this roughly consists in steps 4, 5, and 6. For illustration purposes, we will also be using the agents from the [Hello World agent service](../hello_world_agent_service.md) and we will create a new (but functionally equivalent) "Hello World 2 agent service". To complete all the steps in this guide, you should have a [Görli testnet](https://goerli.net/) wallet address (e.g., [Metamask](https://metamask.io/)) with some GörliETH funds in it.
+
+##Step-by-step instructions
+
+Before starting this guide, ensure that your machine satisfies the [framework requirements](./quick_start.md#requirements) and that
+you have followed the [setup instructions](./quick_start.md#setup). As a result you should have a Pipenv workspace folder.
+
+1. **Identify the IPFS hash of the agent.** This can be some agent with the desired functionality for which you already know the hash, or you can browse it in the repository of a published agent. For this example, we consider the `hello_world` agent, whose hash is
+
+    ```
+    valory/hello_world:0.1.0:bafybeicealdcbxjdejskntddizntwqmlpyxa2ujaxnw2cgy73x3swldwcq
+    ```
+
+    If you want, you can browse the agent contents stored in the IPFS [here](https://gateway.autonolas.tech/ipfs/bafybeicealdcbxjdejskntddizntwqmlpyxa2ujaxnw2cgy73x3swldwcq/hello_world/).
+
+    !!!note
+        Future releases of the {{open_autonomy}} framework will provide convenient commands to browse and discover existing registered components. For now, we assume that we already know the IPFS hash of the agent.
+
+2. **Create the service definition.** Create an empty folder with the service name (e.g., `hello_world_2_service`). Inside that folder, create a `README.md` file, where you can write a description of the service, and a service definition file `service.yaml`, where the agent IPFS hash must be specified:
+
+    ```bash
+    mkdir hello_world_2_service
+    cd hello_world_2_service
+    touch README.md
+    touch service.yaml
+    ```
+
+    !!!info "The structure of the service definition file"
+        As its name suggests, the service definition file `service.yaml` is where
+        the parameters of the agent service are defined, including the particular agent that composes the service. Here is an example of a `service.yaml` file:
+        ```yaml
+        name: hello_world_2_service
+        author: your_name
+        version: 0.1.0
+        description: This is the Hello World 2 service.
+        aea_version: '>=1.0.0, <2.0.0'
```

Comment:
can you confirm this version range is supported?

Reply:
This is copied from hello-world 1, so I assume so.